test/box-tap/check_merge_source.c

```diff
@@ -1,9 +1,10 @@
 #include <lua.h>
 #include <lauxlib.h>
-#include "lib/core/diag.h"
-#include "fiber.h"
 #include "lua/utils.h"
 #include "box/merger.h"
+#include "box/error.h"
+#include "lib/core/diag.h"
```

Comment:
You don't need `lib/core` prefix. (`fiber.h` is in the same folder one line below and works fine.)

Reply:
This was an unnecessary diff. Now I left include `lib/core/diag.h` as it was.